libreyolo/label/assist.py

```
@@ -0,0 +1,290 @@
+        try:
+            import importlib.util
+
+            # LocateAnything needs the VLM extra (its remote-code deps), not just
+            # transformers; decord is the distinctive one. Gate on it so the option
+            # isn't advertised in plain `label`/`sam` installs where it fails to load.
+            return all(importlib.util.find_spec(m) is not None
+                       for m in ("transformers", "decord"))
+        except Exception:  # noqa: BLE001
+            return False
+
+    def _get_la(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_get_la`

(ANN202)